src/screens/ui/AppletBottomTabNavigator.tsx

```diff
@@ -25,10 +29,20 @@ const AppletBottomTabNavigator = ({ route, navigation }: Props) => {
 
   const { title, appletId } = route.params;
 
-  const { data: appletTheme } = useAppletDetailsQuery(appletId, {
-    select: o => mapThemeFromDto(o.data.result.theme),
+  const { data: applet } = useAppletDetailsQuery(appletId, {
+    select: o => AppletModel.mapAppletDetailsFromDto(o.data.result),
   });
 
+  const appletTheme = applet?.theme;
+
+  const appletVersions = useMemo(
+    () =>
+      applet ? [{ appletId: applet!.id, version: applet!.version }] : null,
+    [applet],
+  );
+
+  useCompletedEntitiesQuery(getMonthAgoDate(), appletVersions);
```

Comment:
also, re. getMonthAgoDate, let's discuss it later. Maybe it'd be enough to get values only for the last week. It'd probably ignore the reminder logic for montly events.
But in this situation the amount of records can be huge. Maybe the server should consider and exclude records for the current device id?

Reply:
@anq83  addressing you last question:
I mentioned earlier that the API request returns only actual information of completed entities. It does not contain all the history of completed entities. So there won't be many of them.

Yes, if you say so we can just ask for the last week's values. I just thought that as we have a Monthly event then it will make sense to request data for the last month.